streamwebs_frontend/streamwebs/templates/streamwebs/navigation.html

```diff
@@ -0,0 +1,18 @@
+<div id="topbar">
+    <div class="navbar">
+        <div class="navbar right" align="right"></div>
+        <div class="navbar content"
+             align="right">
+        <nav>
+        <a href="{% url 'streamwebs:sites' %}">Sites</a>
+        <a href="{% url 'streamwebs:index' %}">Resources</a>
+        {% if user.is_authenticated %}
+            <a href="{% url 'streamwebs:logout' %}">Logout</a>
+        {% else %}
+            <a href="{% url 'streamwebs:login' %}">Login</a>
+            <a href="{% url 'streamwebs:register' %}">Create Account</a>
+        {% endif %}
+        </div>
+        </nav>
```

Comment:
Make sure you indent the inner tags, so we can easily see which elements are children of another element.